packages/xrpl/src/client/index.ts

```diff
@@ -126,6 +95,28 @@ export interface ClientOptions extends ConnectionUserOptions {
   timeout?: number
 }
 
+type RequestNextPageType =
```

Comment:
I think it's worth adding a reminder to keep these two types in sync
```suggestion
// Make sure to update both this and `RequestNextPageReturnMap` at the same time
type RequestNextPageType =
```